src/main/java/model/LottoRank.java

```diff
@@ -0,0 +1,43 @@
+package model;
+
+public enum LottoRank {
+
+    FIFTH(3, BonusBallCondition.NON_REGARD, 5000),
+    FORTH(4, BonusBallCondition.NON_REGARD, 50000),
+    THIRD(5, BonusBallCondition.MISMATCH, 1500000),
+    SECOND(5, BonusBallCondition.MATCH, 30000000),
+    FIRST(6, BonusBallCondition.NON_REGARD, 2000000000);
```

Comment:
이렇게 숫자의 자리수가 많아진다면 `_`로 자리수를 구분해 줄 수 있다는 아주 미세한 팁 하나 드리고 갑니다!!!
```suggestion
    FIRST(6, BonusBallCondition.NON_REGARD, 2_000_000_000);
```

Reply:
이렇게 하면 쉼표 구분도 수월해 지겠네요 감사합니다 ㅎㅎ